### DISCUSSION TITLE
I am going to record a video about me coding Tetris like it is 1957 using Fortran and assembly language. What Assembler, emulator, and Fortran compiler should I use for this video?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#139322] [Programming Help]
### Body

I am going to record a video about me coding Tetris like it is 1957 using Fortran and assembly language. What Assembler, emulator, and Fortran compiler should I use for this video?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by phuyalgaurav** [comment]
Maybe gfortran for fortran compiler.
NASM is new but i think would be "old enough" for you to make a video with.
DOSBox is probably the only emulator I can think of for this.

Also use VIM or Emacs to mimic the old coding style, Maybe? idrk what your plans are though

**Comment by ghost** [comment]
What if I also do everything on DOSBox, so I can emulate the environment more accurately?

**Comment by phuyalgaurav** [comment]
Yes possibly. I mean, it's your creative liberty on what to do.
